setup.cfg

```diff
@@ -58,12 +58,9 @@ install_script = scripts/rpm-install.sh
 [flake8]
 exclude = build,.git,.tox,./tests/.env
 ignore = W504,W601
-max-line-length = 119
+max-line-length = 88
 
 [isort]
```

Comment:
You could move the configuration to `pyproject.toml`, which isort documents as a preferred format: https://pycqa.github.io/isort/docs/configuration/config_files.html

Example:
https://github.com/adamchainz/django-htmx/blob/a61c05319b43309716b4b494809e5290c2ae8dfe/pyproject.toml#L8-L9

Reply:
I agree to move `isort` settings into `pyproject.toml`

Reply:
I'll share a different view. 🙂

Flake8 doesn't (and doesn't look like it will, for now) support, `pyproject.toml` so the linting config will be spread over two files. The OCD in me likes having it all in one place. 

Appreciate this is all somewhat subjective... So let's move it to `tox.ini`? 😁

Reply:
I know `flake8` will not support `pyproject.toml` so move to `flake9`
https://pypi.org/project/flake9/

> Flake9 is a fork of Flake8 that supports reading configuration from a pyproject.toml file, because the authors of Flake8 won’t implement it.
> In addition, because it’s 2020, Python 2.7 support has been dropped.

Reply:
flake9 is a hostile (unmaintained) fork. It's not that "the authors [maintainer] of flake8 won't implement it", it's that the config system in flake8 is already overcomplicated and they want to tidy it before adding pyproject.toml support: https://github.com/PyCQA/flake8/issues/234

Reply:
> flake9 is a hostile (unmaintained) fork ...

I was hoping it was clear from the smile at the end of the sentence that I wasn't really proposing to use `flake9`, it was a joke. 😅️

I was trying to play down @smithdc1 worry and my personal frustration of having to keep the `setup.cfg` file in my projects only for the `flake8` configurations, even if I know their reasons for not having implemented `pyproject.toml` yet